helm-charts/interoperator/values.yaml

```diff
@@ -28,6 +28,17 @@ broker:
     serviceDomain: https://my-tenant-onboarding.com
     username: quota_user
     password: quota_password
+    regions:
+      eu10:
+        oauthDomain: https://myauth-domain.com
+        serviceDomain: https://my-tenant-onboarding.com
+        username: quota_user
+        password: quota_password
+      eu20:
+        oauthDomain: https://myauth-domain.com
+        serviceDomain: https://my-tenant-onboarding.com
+        username: quota_user
+        password: quota_password
```

Comment:
Probably a good idea to comment out this section similar to image pull secrets in this file

Reply:
Done